backend/Virkarekisteri/src/Functions/Employees/GetPositionEmployee.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Virkarekisteri.Repositories;
+
+namespace Virkarekisteri.Functions.Employees;
+
+public class GetPositionEmployee(
+    ILogger<GetPositionEmployee> logger,
+    IPositionEmployeeRepository positionEmployeeRepository
+)
+{
+    /// <summary>
+    /// Gets a position employee by id
+    /// </summary>
+    /// <param name="req">Input GET request containing the ID</param>
+    /// <returns>
+    /// The position employee with the given ID
+    /// </returns>
+    [Function("GetPositionEmployee")]
```

Comment:
You need to add the required authorization attribute here

Reply:
done